### ISSUE TITLE
API to report the state of autonomy

### ISSUE THREAD

**Issue by tsaoyu** [WRSC/tracking#48]
Can we have an API to report the state of each autonomous ship to the server? 

- Autonomous 
- Radio controlled (when the sailing boat is controlled by human)
- On transit (when the sailing boat is carried by the safety boat)

So the server can filter out those way points are not made in autonomous mode.

**Comment by takluyver** [comment]
Would boats need to send this information in real time? It would be a substantial step up in requirements if the boat's control system has to be connected to the tracking system.

Recording when it's in transit would presumably involve someone pressing a button to record when that state changes; it would be easy to forget to do this while you're focused on getting the boat in/out the water. So we'd need some way to correct it later.

**Comment by tsaoyu** [comment]
No, the boat doesn't have to send the state to the tracking system. This API is for the referee or the safety boat driver to report the state using an app (or on a page in tracking system #49). 

It is good to have a way to correct the record. I will try to find few local Ruby on rails developers and see what is the best way to do this.
